index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio page</title>
+    <header>
+      <!-- The Header section -->
+      <div class="header">
+          <a href="http://127.0.0.1:5500/signup.html">Login!</a>
```

Comment:
you cannot set ip adress as the url of your redirected page. it will not work. you should redirect it properly to the resource you want to show/navigate to.